backend/.env

```diff
@@ -1,6 +1,7 @@
 REST_API_PORT=5000
 API_DOCS_ENDPOINT_URL=/rest-api-docs
 GOOGLE_CLIENT_ID=1052788207529-fjnskiqrsm09i9kbujp3gmtasnp21su7.apps.googleusercontent.com
+JWT_SECRET_KEY=MOVE_IT_TO_GITHUB_SECRETS
```

Comment:
Jaka to ma byc wartosc :) ? Napisz mi gdzies na Slacku to dodam.

Reply:
chyba obojętne, potrzebuję tylko wiedzieć pod jaką nazwą to zapisałeś, bo rozumiem, że jak w github secrets zapiszesz to pod nazwą JWT_SECRET_KEY to wtedy w pliku backend.yml na dole pliku w env: pod HD_MONGO_CONNECTION_STRING dajesz w kolejnej lini
`JWT_SECRET_KEY: ${{secrets.JWT_SECRET_KEY}}`
i potem bezpośrednio w kodzie w pliku .ts możesz się do tego odwołać jako `process.env.JWT_SECRET_KEY`? Czy jednak coś źle rozumiem?

+ tak samo chyba by się przydało zrobić z GOOGLE_CLIENT_ID, hmmm?

Reply:
Chyba musi byc HD_ najpierw, dla Heroku.
To napisz mi jakie klucz - wartosc maja byc :) Bo nie mamy wartosci tutaj.